pages/table/sticky-columns.page.tsx

```diff
@@ -0,0 +1,318 @@
+  {
+    id: 'variable',
+    header: 'Simple table',
+    minWidth: 200,
+    cell: item => {
+      return item.name;
+    },
+  },
+  {
+    id: 'description',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-2',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-3',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-4',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-5',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-6',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-7',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-8',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-9',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-10',
+    header: 'Description',
+    cell: item => <Link href="#">Link: {item.description}</Link> || '-',
+  },
+  {
+    id: 'description-11',
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+  {
+    id: 'description-12',
+    minWidth: 200,
+    header: 'Description',
+    cell: item => item.description || '-',
+  },
+];
+
+const ITEMS = [
+  {
+    name: 'Item 1',
+    alt: 'First',
+    description: 'This is the first item',
+    type: '1A',
+    size: 'Small',
+  },
+  {
+    name: 'Item 2',
+    alt: 'Second',
+    description: 'This is the second item',
+    type: '1B',
+    size: 'Large',
+  },
+  {
+    name: 'Item 3',
+    alt: 'Third',
+    description: '-',
+    type: '1A',
+    size: 'Large',
+  },
+  {
+    name: 'Item 4',
+    alt: 'Fourth',
+    description: 'This is the fourth item',
+    type: '2A',
+    size: 'Small',
+  },
+  {
+    name: 'Item 5',
+    alt: '-',
+    description: 'This is the fifth item with a longer description',
+    type: '2A',
+    size: 'Large',
+  },
+  {
+    name: 'Item 6',
+    alt: 'Sixth',
+    description: 'This is the sixth item',
+    type: '1A',
+    size: 'Small',
+  },
+];
+
+export default () => {
+  const [preferences, setPreferences] = React.useState<CollectionPreferencesProps.Preferences>({
+    stickyColumns: { first: 1, last: 1 },
+  });
+
+  return (
+    <ScreenshotArea>
+      <h1>Sticky columns</h1>
+      <SpaceBetween size="xl">
+        <Table
+          data-test-id="simple"
+          stickyColumns={{ first: 1, last: 1 }}
+          columnDefinitions={COLUMN_DEFINITIONS}
+          items={ITEMS}
+          sortingDisabled={true}
+          ariaLabels={{
+            tableLabel: 'simple',
+          }}
+          header={<Header>Simple table</Header>}
+        />
+        <Table
+          data-test-id="resizable"
```

Comment:
The padding between the right-most resize handle and the table's border look very small. Is that expected?

<img width="1438" alt="Screenshot 2023-04-18 at 14 22 21" src="https://user-images.githubusercontent.com/20790937/232776026-f0433135-c5a0-4fb8-b432-42565459aa93.png">

Reply:
When the table wrapper is in focus that resize handle touches the focus outline.